src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/Formatters/ILogMessageFormatter.cs

```diff
@@ -280,7 +280,7 @@ public void Serialize(ref MessagePackWriter writer, ILogMessage? value, MessageP
             }
         }
 
-        private void SerializeCommonProperties(ref MessagePackWriter writer, ILogMessage value, MessagePackSerializerOptions options)
+        private static void SerializeCommonProperties(ref MessagePackWriter writer, ILogMessage value, MessagePackSerializerOptions options)
```

Comment:
There was a CA1822 (Mark member as Static) analyzer warning here, but suppression didn't resolve the error, hence fixed the warning directly.